packages/dialtone-vue2/recipes/comboboxes/combobox_multi_select/combobox_multi_select.vue

```diff
@@ -23,9 +23,9 @@
           class="combobox__chip-wrapper"
         >
           <dt-chip
-            v-for="item in selectedItems"
+            v-for="(item, index) in selectedItems"
             ref="chips"
-            :key="item"
+            :key="`${item}-${index}`"
```

Comment:
Can we use getUniqueString function for this instead? We should avoid using index as a key in a v-for?

Reply:
yes sure, let me update that

Reply:
Hm my current implementation (easiest one) will not work as`getUniqueString` is being calculated more than one time per item and that is causing several issues (not focus properly, remove with keyboard not working, etc)
![image](https://github.com/dialpad/dialtone/assets/144126346/0e21bbe5-1ee9-41c5-b858-2e058cba4368)
(thanks dev for writing tests, I wouldn't detect this issue without them).

so I was testing different options.. 
1) creating a new computed prop like this 
```
 selectedItemsWithKeys () {
      return this.selectedItems.map(item => ({
        value: item,
        uniqueKey: utils.getUniqueString(item),
      }));
    }, 
```
and then use that for the array, but seems like an overkill for this minor issue..
2) using index
3) allowing client to send an object array instead of string array and then using that key.

Can I ask for an opinion here? What do you think would be better? If you have a better idea please let me know

Reply:
> 1. creating a new computed prop like this

I think this one will work best. It avoids a breaking change and makes it easiest for the consumer of the component to use.